File: 1149.py
y = input().split(" ")
a = int(y[0]); n = int(y[-1])
i = 0
if a > n:
    for l in range(a, n+a):
        i+=l
else:
    for p in range(n, n+a):
        i+=p
print(i)

File: 1149.java
import java.util.Scanner;
public class Main {
public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int a = input.nextInt(), n = input.nextInt(), i = 0;
        while(n<=0){
                n = input.nextInt();
        }
        if(a > n) {
                for(int l = a; l < n+a; l++) {
                        i += l;
                }
        }
        else {
                for(int p = n; p < n+a; p++) {
                        i += p;
                }
        }
        System.out.println(i);
}
}
